Sources/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewLayout.swift

```diff
@@ -3,47 +3,19 @@
 //
 
 import Foundation
+import StreamChat
 import UIKit
 
-open class ChatChannelListCollectionViewLayout: UICollectionViewFlowLayout {
-    override public init() {
-        super.init()
-        commonInit()
-    }
-    
-    public required init?(coder: NSCoder) {
-        super.init(coder: coder)
-        commonInit()
-    }
-        
+open class ChatChannelListCollectionViewLayout: ListCollectionViewLayout {
+
     override open func prepare() {
         super.prepare()
-        
+
+        // We should always propose the full width of the collection view for the cell width
         estimatedItemSize = .init(
             width: collectionView?.bounds.width ?? 0,
-            height: 64
+            height: estimatedItemSize.height
         )
```

Comment:
This can live in `ListCollectionViewLayout`

Reply:
I guess then we don't actually need `ChatChannelListCollectionViewLayout` anymore right? Will delete it

Reply:
@nuno-vieira True 👍